homeassistant/components/fritzbox/climate.py

```diff
@@ -53,7 +52,7 @@
 
 
 async def async_setup_entry(
-    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: Callable
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
```

Comment:
You replace a name with another name. Is there any added value?

Reply:
Well, arguments of the callback are now type checked. Before it it was any number of `Any` and return value was `Any` as well. Now it's the precise type.